### PR TITLE
fix(community): restore missing contact icons and adjust existing ones #173

### LINKED ISSUE #173: Add Geyser to socials.
Tag is `contact:geyser`.

### PR DESCRIPTION
**Pull request by dadofsambonzuki**
## Summary

This PR restores missing contact icons that were accidentally removed in a previous commit, adds the geyser icon, updates the matrix icon visibility, and fixes a security issue with URL sanitization.

Also fixes #173 

### 1. chore(prettier): ignore social icons from formatting checks
- Update `.prettierignore` to exclude social icons directory

### 2. feat(community): add geyser icon  
- Add official Geyser brand image (`geyser.jpg`)

### 3. feat(community): update matrix icon with brand color and padding
- Change fill from `currentColor` (transparent) to Matrix brand color `#0DBD8B`
- Add padding via viewBox to display properly in circular containers

### 4. feat(community): add type definitions for eventbrite, reddit, simplex contact fields
- Add `contact:eventbrite`, `contact:reddit`, `contact:simplex` to AreaTags type

### 5. feat(community): restore eventbrite, reddit, simplex social links
- Add props and UI rendering for restored social links in Socials.svelte

### 6. feat(community): integrate eventbrite, reddit, simplex fields in components
- Update CommunityCard.svelte, AreaPage.svelte, and map page to use new fields

### 7. fix(utils): prevent relative URLs in sanitizeUrl by removing base URL
- **Security fix**: Remove base URL from URL constructor to reject relative URLs
- Previously allowed `reddit.com/r/foo` to be treated as safe (would create broken relative links)
- Now only accepts absolute URLs with safe protocols

## Testing

- [x] Verify geyser icon displays correctly
- [x] Verify eventbrite icon displays correctly  
- [x] Verify reddit icon displays correctly
- [x] Verify simplex icon displays correctly
- [x] Verify matrix icon is now visible (green with padding)
- [x] Verify sanitizeUrl rejects relative URLs like `reddit.com/r/foo`
- [x] Verify sanitizeUrl accepts absolute URLs like `https://reddit.com/r/foo`

🤖 Generated with opencode

## Summary by CodeRabbit

* **New Features**
  * Communities can now display Eventbrite, Reddit, and Simplex contact links in social/contact areas.
* **Bug Fixes**
  * Improved URL handling: scheme-less addresses are normalized to https:// and only safe protocols are allowed to prevent invalid or unsafe links.
* **UI**
  * Replaced one social image asset to improve display quality.
* **Tests**
  * Added extensive tests covering URL normalization, validation, and edge cases.

### CODE REVIEW

File: src/lib/utils.ts [hunk 1]
```diff
@@ -24,15 +24,16 @@ export const isValidPlaceId = (id: string): boolean => /^(\d+|(?:node|way|relati
...
 	try {
-		const parsed = new URL(url, 'https://btcmap.org');
+		const parsed = new URL(url);
 		if (SAFE_URL_PROTOCOLS.includes(parsed.protocol.toLowerCase())) {
 			return url;
 		}
 	} catch {
-		// Invalid URL
+		// Invalid URL or relative URL (new URL() throws for relative URLs without base)
 	}
```
[the remark this answers is not in the document]
Comment (dadofsambonzuki) [reply src/lib/utils.ts]: Dupe.

**Review by escapedcat — APPROVED** [review]
